src/analysis/thermo/thermoFBA/fastSNP.m

```diff
@@ -0,0 +1,135 @@
+function [N, resInfo] = fastSNPcobra(model, varargin)
```

Comment:
This function does actually not correspond to the name of the file.

Reply:
I will fix this separately.